crates/rpc/rpc-types/src/eth/engine.rs

```diff
@@ -40,9 +40,9 @@ impl From<SealedBlock> for ExecutionPayload {
             .body
             .iter()
             .map(|tx| {
-                let mut encoded = BytesMut::new();
+                let mut encoded = Vec::default();
```

Comment:
Vec::default looks weird lolg
```suggestion
                let mut encoded = Vec::new();
```